### ISSUE TITLE
Unable to generate client & server

### ISSUE THREAD

**Issue by msalman86** [grpc/grpc-swift#59]
hey,

I am not able to generate client & server code when I run

```
protoc ./helloworld.proto --swift_out=./generated/  --swiftgrpc_out=./generated/
```

I am using the latest github `master`. This is generating `helloworld.pb.swift` and `swiftgrpc.log`. Do I need to use an additional param? 

Any help would be much appreciated. Thanking in advance!

P.S. `swiftgrpc.log` is attached for reference
[swiftgrpc.log.zip](https://github.com/grpc/grpc-swift/files/1066657/swiftgrpc.log.zip)

**Comment by timburks** [comment]
Hi, if you're getting swiftgrpc.log, then the swiftgrpc plugin is being run. Would you mind sharing your helloworld.proto file with me? From the log it appears that the plugin is looking in google/protobuf/empty.proto for services to generate and not finding any... it should be looking in helloworld.proto instead.

**Comment by msalman86** [comment]
Thanks for replying back. The proto file looks like this

```
syntax = "proto3";

import "google/protobuf/empty.proto";

service Greater {

  rpc SayHelloWorld (google.protobuf.Empty) returns (HelloReply) {}

  rpc SayHelloName (HelloName) returns (HelloReply) {}
}

message HelloName {
   string name = 1;
}

message HelloReply {
  string message = 1;
}

```

**Comment by timburks** [comment]
If that's the exact file that you used, you might be missing a "package" statement. I added one to create the following `helloworld.proto`:
```
syntax = "proto3";

package helloworld;

import "google/protobuf/empty.proto";

service Greater {

  rpc SayHelloWorld (google.protobuf.Empty) returns (HelloReply) {}

  rpc SayHelloName (HelloName) returns (HelloReply) {}
}

message HelloName {
   string name = 1;
}

message HelloReply {
  string message = 1;
}
```

Then I compiled that file with the following command:
`protoc helloworld.proto --swift_out=. --swiftgrpc_out=.`

That produced these three Swift files:
`helloworld.client.pb.swift helloworld.pb.swift helloworld.server.pb.swift`

I think that explains your problem. When I run without the package declaration, I see the same problem that you reported.

(Sorry about the lack of proper error handling in the swiftgrpc plugin, I'll fix that)
